tests/Microsoft.Identity.Web.Test/Certificates/DefaultCertificateLoaderTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.Identity.Web.Test.Common;
+using Xunit;
+
+namespace Microsoft.Identity.Web.Test.Certificates
+{
+    public class DefaultCertificateLoaderTests
+    {
+        // https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationMenuBlade/Credentials/appId/86699d80-dd21-476a-bcd1-7c1a3d471f75/isMSAApp/
+        [InlineData(CertificateSource.Base64Encoded, null, TestConstants.CertificateX5c)]
+        //[InlineData(CertificateSource.KeyVault, TestConstants.KeyVaultContainer, TestConstants.KeyVaultReference)]
```

Comment:
```suggestion
        // [InlineData(CertificateSource.KeyVault, TestConstants.KeyVaultContainer, TestConstants.KeyVaultReference)]
```

Reply:
i'm not sure what to do w/these tests yet, so not taking these suggestions atm.

---
In reply to: [436370379](https://github.com/AzureAD/microsoft-identity-web/pull/190#discussion_r436370379) [](ancestors = 436370379)